experimental/gpnf-gv-duplicate-child-entries.php

```diff
@@ -14,6 +14,13 @@
  * As an example, this snippet is configured to duplicate child entries for Nested Form fields 1 and 2 in form 17.
  */
 add_action( 'gravityview/duplicate-entry/duplicated', function ( $duplicated_entry, $original_entry ) {
+	gpnf_duplicate_child_entries( $duplicated_entry, $original_entry );
+}, 10, 2 );
+add_action( 'gpeb_entry_duplicated', function ( $original_entry ) {
```

Comment:
Can you add a newline in between lines 18 and 19? That will help with readability.

Reply:
Thanks @claygriffiths! Done.